resources/views/includes/header.blade.php

```diff
@@ -8,7 +8,7 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="/">{{ config('app.name', 'HRM') }}</a>
+      <a class="navbar-brand" href="/admin">{{ config('app.name', 'HRM') }}</a>
```

Comment:
Use `route('home')` instead of statically writing the admin route. Check this everywhere throughout the code and fix accordingly.